src/main/java/io/rcktapp/api/handler/firehose/FirehoseDb.java

```diff
@@ -97,6 +99,31 @@ public void bootstrapApi() throws Exception {
         }
     }
 
+    private List<String> listAllDeliveryStreamNames() {
+
+        ListDeliveryStreamsResult listDeliveryStreamsResult;
+        List<String> deliveryStreamNames = new ArrayList<>();
+
+        do {
+            String last = deliveryStreamNames.stream().reduce((first, second) -> second).orElse(null);
+
+            ListDeliveryStreamsRequest listDeliveryStreamsRequest = getRequest(last);
```

Comment:
the 'last' variable is used exactly once.  Unless you're doing it this way for readability (a reasonable argument), it should be inlined.

Reply:
Yeah I was also on the fence about inlining. Leaving it, pending style guide/coding conventions.